<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ticketWindow</class>
 <widget class="QDialog" name="ticketWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="categorieBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>Problèmes logiciels</string>
   </property>
   <item>
    <property name="text">
     <string>Problèmes logiciels</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Informations</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Aide Paiement</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Catégorie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Envoyer</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>681</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="titreTicket">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
